### PR TITLE
Create how-k8s-deploy-cn.md

### PR DESCRIPTION
**Pull request by cyxinda**
在k8s上面搭建apisix网关，供大家参考！

NOTE: Please read the Contributing.md guidelines before submitting your patch:

https://github.com/apache/incubator-apisix/blob/master/Contributing.md#how-to-add-a-new-feature-or-change-an-existing-one

### Summary

SUMMARY_HERE

### Full changelog

* [Implement ...]
* [Add related tests]
* ...

### Issues resolved

Fix #XXX

### CODE REVIEW

**Comment by membphis** [comment]
@cyxinda please take a look at the output of CI

**Comment by dabue** [comment]
I got the error when I use the healthcheck to get the status of upstrems.  Has anyone ever had the same problem?

Error:
2020/03/31 17:03:01 [error] 47#47: *128805375 failed to run balancer_by_lua*: /usr/local/openresty/lualib/resty/lock.lua:153: API disabled in the context of balancer_by_lua*
stack traceback:
        [C]: in function 'sleep'
        /usr/local/openresty/lualib/resty/lock.lua:153: in function 'lock'
        /opt/app/mdcs//deps/share/lua/5.1/resty/healthcheck.lua:207: in function 'locking_target_list'
        /opt/app/mdcs//deps/share/lua/5.1/resty/healthcheck.lua:1315: in function 'new'
        /opt/app/mdcs//lua/travelsky/http/checker.lua:25: in function 'create_obj_fun'
        /opt/app/mdcs//lua/apisix/core/lrucache.lua:50: in function 'lrucache_checker'
        /opt/app/mdcs//lua/travelsky/http/checker.lua:35: in function 'fetch_checker'
        /opt/app/mdcs//lua/travelsky/http/balancer.lua:221: in function 'pick_server'
        /opt/app/mdcs//lua/travelsky/http/balancer.lua:287: in function 'load_balancer'
        /opt/app/mdcs//lua/travelsky.lua:259: in function 'http_balancer_phase'
        balancer_by_lua:2: in main chunk while connecting to upstream, client: 101.5.19.12, server: , request: "POST / HTTP/1.1", host: "101.5.19.15:8080"



88786912@qq.com
 
From: YuanSheng Wang
Date: 2020-03-31 21:58
To: apache/incubator-apisix
CC: Subscribed
Subject: Re: [apache/incubator-apisix] Create how-k8s-deploy-cn.md (#1376)
@cyxinda please take a look at the output of CI

File: doc/how-k8s-deploy-cn.md [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+## 本地安装apisix
+- 使用yum安装
+```bash
+yum install -y https://github.com/apache/incubator-apisix/releases/download/1.1/apisix-1.1-0.el7.noarch.rpm
+```
+## 本地安装dashboard
+参见：[https://github.com/apache/incubator-apisix/tree/v1.1](https://github.com/apache/incubator-apisix/tree/v1.1) -- dashboard安装部分
+或者 [https://blog.csdn.net/cyxinda/article/details/105099571](https://blog.csdn.net/cyxinda/article/details/105099571)--dashboard安装部分
+## 工作目录结构：
+如下是最终生成的工作目录
```
Comment (membphis) [inline doc/how-k8s-deploy-cn.md:12]: add a blank line before this line, and please fix other similar points

File: doc/how-k8s-deploy-cn.md [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+```python
```
Comment (membphis) [inline doc/how-k8s-deploy-cn.md:13]: we should use ```shell here

File: doc/how-k8s-deploy-cn.md [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+	```bash
+	cp ../incubator-apisix-docker-master/example/apisix_conf/config.yaml .
+	```
+ 2. 拷贝步骤【本地安装dashboard】中安装好的dashboard
+	```bash
+		cp -r /usr/local/apisix/dashboard .
+	```	
+- 修改apisix的配置文件
+	```yaml
+	apisix:
```
Comment (membphis) [inline doc/how-k8s-deploy-cn.md:116]: we should use space instead of `tab`

**Comment by moonming** [comment]
@cyxinda ping

**Pull request closed without merging**